sopel/plugin.py

```diff
@@ -615,25 +615,42 @@ def add_attribute(function):
     return add_attribute
 
 
-def intent(*intent_list):
-    """Decorate a callable to trigger on a message with any of the given intents.
+def ctcp(function=None, *command_list):
+    """Decorate a callable to trigger on CTCP commands (mostly, ``ACTION``).
 
-    :param str intent_list: one or more intent(s) on which to trigger (really,
-                            the only useful value is ``action``)
+    :param str ctcp_command: one or more CTCP command(s) on which to trigger
```

Comment:
The param's name is `command_list` though? This is how we've done it for other variable-length decorators like `event` above.

```suggestion
    :param str command_list: one or more CTCP command(s) on which to trigger
```

Reply:
My mistake!